docs/howto/run_material_mapping.rst

```diff
@@ -87,6 +87,58 @@ Then edit the config-map.json file
 
    $ python3 <source>/Examples/Scripts/MaterialMapping/configureMap.py geometry-map.json config-map.json
 
+Geometry visualisation and preparation of JSON files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
In the Howtos section  there is this "[Howto produce geometry selection for SP making](https://acts.readthedocs.io/en/latest/howto/produce_geometry_selection.html)"
I guess this doc kind of makes the other obsolete. Should it be removed then in this PR?

Reply:
At the very list they should be cross-linked, but yeah maybe it's best to remove that one.